app/app.go

```diff
@@ -180,10 +180,20 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+	// Try to get CELESTIA_HOME env variable, if not set, use user home dir
+	userHomeDir = getEnv("CELESTIA_HOME", userHomeDir)
 
 	DefaultNodeHome = filepath.Join(userHomeDir, "."+Name)
 }
 
+// Get environment variable or fallback to default value
+func getEnv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
```

Comment:
As far as I know, an empty string is the same as an environment variable that is not set. So how can we catch this?

Reply:
> also new method requires new unit test :-)
> 
> We should right the unit test in a way that it is set up to be extended for all future env vars.

Wo can assist with that? I don't have a lot of experience with Go.

Reply:
plus then you can skip any new unit tests :-)

Reply:
Great idea! Thanks :)
I needed to adapt your solution a little because I had an error: https://github.com/celestiaorg/celestia-app/pull/1473/commits/d0273abd439fbf3787a3fdebd4630982de3d1b54